File: beecrowd/1010.cpp
// created by mostafizzfuad
#include <bits/stdc++.h>
using namespace std;
int main() {

    int codeOfProduct1, codeOfProduct2, unitOfProduct1, unitOfProduct2;
    double perUnitPrice1, perUnitPrice2;

    cin >> codeOfProduct1 >> unitOfProduct1 >> perUnitPrice1 >> codeOfProduct2 >> unitOfProduct2 >> perUnitPrice2;

    double total = (unitOfProduct1 * perUnitPrice1) + (unitOfProduct2 * perUnitPrice2);

    cout << "VALOR A PAGAR: R$ " << fixed << setprecision(2) << total << endl;
    
}

File: beecrowd/1010.cs
// created by mostafizzfuad
using System;
class URI {
    static void Main(string[] args) {

        int codeOfProduct1 = Convert.ToInt32(Console.ReadLine());
        int unitOfProduct1 = Convert.ToInt32(Console.ReadLine());
        double perUnitPrice1 = Convert.ToDouble(Console.ReadLine());

        int codeOfProduct2 = Convert.ToInt32(Console.ReadLine());
        int unitOfProduct2 = Convert.ToInt32(Console.ReadLine());
        double perUnitPrice2 = Convert.ToDouble(Console.ReadLine());

        double total = unitOfProduct1 * perUnitPrice1 + unitOfProduct2 * perUnitPrice2;

        Console.WriteLine("VALOR A PAGAR: R$ " + total.ToString("0.00"));

    }
}


// runtime error